webdriver-spec.html

```diff
@@ -1435,288 +1435,513 @@
-      named <code>httpProxy</code> from <var>proxy</var>.
+<ol>
+<li><p>Let <var>matched capabilities</var> be a JSON Object with the following
+entries:
 
-     <li><p>Let <var>http proxy port</var> be the result of <a>getting a property</a>
-      named <code>httpProxyPort</code> from <var>proxy</var>.
+  <dl>
+  <dt>"<code>browserName</code>"
+  <dd>Lowercase name of the user agent.
 
-     <li><p>Let <var>ssl proxy</var> be the result of <a>getting a property</a>
-      named <code>sslProxy</code> from <var>proxy</var>.
+  <dt>"<code>browserVersion</code>"
+  <dd>The user agent version, as a string.
 
-     <li><p>Let <var>ssl proxy port</var> be the result of <a>getting a property</a>
-      named <code>sslProxyPort</code> from <var>proxy</var>.
+  <dt>"<code>platformName</code>"
+  <dd>Lowercase name of the current platform.
 
-     <li><p>Let <var>socks proxy</var> be the result of <a>getting a property</a>
-      named <code>socksProxy</code> from <var>proxy</var>.
+  <dt>"<code>platformVersion</code>"
+  <dd>The platform version, as a string.
 
-     <li><p>Let <var>socks proxy port</var> be the result of <a>getting a property</a>
-      named <code>socksProxyPort</code> from <var>proxy</var>.
+  <dt>"<code>acceptSslCerts</code>"
+  <dd>False, indicating the session <em>will not</em> implicitly trust untrusted
+  or self-signed SSL certificates on <a data-lt=go>navigation</a>.
+  </dl>
 
-     <li><p>If <var>socks proxy</var> and <var>socks proxy port</var> is defined:
+<li><p>Let <var>capabilities length</var> be the number of entries in <var>capabilities</var>.
 
-      <ol>
-        <li><p>Let <var>socks version</var> be the result of <a>getting a property</a>
-         named <code>socksVersion</code> from <var>proxy</var>.
+<li><p>Let <var>k</var> be 0.
 
-        <li><p>Let <var>socks username</var> be the result of <a>getting a property</a>
-         named <code>socksUsername</code> from <var>proxy</var>.
+<li><p>While <var>k</var> &lt; <var>capabilities length</var>:
 
-        <li><p>Let <var>socks password</var> be the result of <a>getting a property</a>
-         named <code>socksPassword</code> from <var>proxy</var>.
-      </ol>
+  <ol>
+  <li><p>Let <var>capability name</var> be the name of the entry at index
+  <var>k</var> in <var>capabilities</var>.
 
-     <li><p>Follow implementation defined steps to set the proxy
-      using defined variables from previous steps.
-      If there are items that can not be set during this process return <a>error</a>.
-    </ol>
+  <li><p>Let <var>capability value</var> be the value of the entry at index
+  <var>k</var> in <var>capabilities</var>.
 
-   <dt>Otherwise
-   <dd><p>Return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
-  </dl>
+  <li><p>Switch on <var>capability name</var>:
+
+    <dl class=switch>
+      <dt>"<code>browserName</code>"
+      <dd><p>If <var>capability value</var> is not equal to
```

Comment:
This step is within an iteration over the set capabilities, so capability value _will_ be set. It _may_ be set to `null`  (JSON does not support `undefined`), but that is implicitly handled by the "not equal to" comparison.